templates/default/sentinel.conf.erb

```diff
@@ -117,6 +117,7 @@ sentinel failover-timeout <%=@name%> <%=@failovertimeout%>
 # Example:
 #
 # sentinel notification-script mymaster /var/redis/notify.sh
+<%= "sentinel notification-script #{@name} #{@notification_script}" unless @notification_script.nil? %>
```

Comment:
Here's the commit notification-script support was added: https://github.com/antirez/redis/commit/baace5fc42dbe527a452d8864f37716d644f444a#diff-9cf0c48812ea7f4e98ef1281e344017f. Looks like Redis 2.6.